components/ui/AppCard.vue

```diff
@@ -70,6 +86,10 @@ export default class AppCard extends Vue {
   get hasTags () {
     return Array.isArray(this.tags) && this.tags.length > 0
   }
+
+  get hasTooltipTags () {
```

Comment:
`hasTags` and `hasTooltipTags` do the same with different variables, you can create an unique method and reuse it passing it the variable